<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>메이플보스분배계산기</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource.qrc">
    <normaloff>:/Icon/i15952174095.ico</normaloff>:/Icon/i15952174095.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>196</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl01">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>전체 분배금액</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTotal">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl02">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1인당 분배 금액(3인)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPer3">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl03">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1인당 분배 금액(4인)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPer4">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCopy">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>복사/캡쳐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCalEnd">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>정산완료</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="font">
         <font>
          <family>넥슨Lv2고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="Table">
        <property name="font">
         <font>
          <family>넥슨Lv2고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="font">
         <font>
          <family>넥슨Lv2고딕</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="4">
         <widget class="QLabel" name="lbl06">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>보스명</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl07">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>아이템 이름</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="lbl09">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>가격</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="cbBossName">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl10">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>파티원수</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lbl08">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>갯수</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="deDate">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbItemName">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="sbCount">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl04">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>날자</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="cbBossLevel">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="sbParty">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lbl05">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>보스 등급</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLineEdit" name="lePrice">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="maxLength">
           <number>13</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="8,2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbInsert">
          <property name="font">
           <font>
            <family>넥슨Lv2고딕</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>입력</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1274</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
